types-doc/bsd-definition.md

```diff
@@ -0,0 +1,49 @@
+<!--  NOTE: Auto-generated from the JSON PURL type definition.
+Do not manually edit this file. Edit the JSON type definition instead. -->
+
+# PURL Type Definition: bsd
+
+- **Type Name:** BSD
+- **Description:** BSD Operating System variant packages
+- **Schema ID:** `https://packageurl.org/types/bitname-definition.json`
+
+## PURL Syntax
+
+The structure of a PURL for this package type is:
+
+    pkg:bsd/<namespace>/<name>@<version>?<qualifiers>#<subpath>
+
+## Repository Information
+
+- **Use Repository:** Yes
+- **Note:** The default repository varies among BSD Operating System variants. The default repositories are http://pkg.freebsd.org for FreeBSD, https://cdn.openbsd.org/pub/OpenBSD for OpenBSD, and https://cdn.netbsd.org/pub/NetBSD for NetBSD.
+
+## Namespace definition
+
+- **Requirement:** Required
+- **Native Label:** vendor
+- **Note:** `The namespace is the "vendor" name such as "freebsd", "openbsd", or "netbsd". It is not case sensitive and must be lowercased.`
+
+## Name definition
+
+- **Native Label:** name
+- **Note:** `The name is the component name. It must be lowercased.`
```

Comment:
This requirement is bit complicated. How about KiCAD or FreeCAD for example whom likes that their name is written in strange way or X packages. There should be possibility to have also uppercase chars and how about is this alpha-numeric and how about packages like `py311-my-python-package3` or `package_name`

Reply:
Is there need to say anything about umlauts or similar stuff? How is UTF-8 things handled or is PURL similar to URL that one can decode inside with URL decoding if it's difficult character?